CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## VERSION 2.1.3
```

Comment:
I think we agreed that we were going to take minor up on this cases, so maybe it should be 2.2.0?